variables.tf

```diff
@@ -218,3 +207,15 @@ variable "fleet_no_proxy" {
   default     = ""
   description = "(optional) hosts or domains to bypass the proxy for fleet traffic"
 }
+
+variable "prometheus_enabled" {
+  type        = bool
+  default     = false
+  description = "(optional) enable Prometheus metrics"
+}
+
+variable "fedramp_mode_enabled" {
+  type        = bool
+  default     = false
+  description = "(optional) enable Fedramp mode"
```

Comment:
Is fedramp something we are making optional for AWS? I thought this was just an azureism

Reply:
Nope, it's for all three cloud environments.